contracts/v0.8/AccountAPI.sol

```diff
@@ -40,7 +40,7 @@ contract AccountAPI {
     }
 
     /// @notice FIXME
```

Comment:
Not linked to this PR but why do we have still `FIXME` here ?

Reply:
The idea was to fill this up in the future. Set the proper documentation for each method.